### PR TITLE
Remove last uses of slog and logentries

### PR DESCRIPTION
**Pull request by cat5inthecradle**
Our subscription to LogEntries.com is up for renewal in July. This PR removes our last remaining use of this service.

## Links

Latest Slack Thread: https://codedotorg.slack.com/archives/C03CK49G9/p1654705538942419
Old Slack Thread: https://codedotorg.slack.com/archives/C0T0PNTM3/p1635530518124200
Older Slack Thread: https://codedotorg.slack.com/archives/C0T0PNTM3/p1578525822013500

Jira Issues:
- https://codedotorg.atlassian.net/browse/INF-360
- https://codedotorg.atlassian.net/browse/LP-2094

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by jamescodeorg** [comment]
I think there's some related configuration in deployment.rb, config.yml.rb, cdo.rb, and env.rb (search for 'slog').

**Review by davidsbailey — APPROVED** [review]
I would suggest a follow-up PR to remove other occurrences of slog:
Screen Shot 2022-06-08 at 10 04 03 AM

**Comment by Hamms** [comment]
Unless there's a specific reason why we want these calls to be removed ASAP, I'd actually suggest we do the work to remove other occurrences of slog in this PR

**Comment by cat5inthecradle** [comment]
> Unless there's a specific reason why we want these calls to be removed ASAP, I'd actually suggest we do the work to remove other occurrences of slog in this PR

Agreed. We can merge if the account gets terminated sooner, but we've got a little bit of time.

File: config/production.yml.erb [hunk 1]
```diff
@@ -11,7 +11,6 @@ redshift_host: !Secret
 redshift_password: !Secret
 redshift_username: !Secret
 sinatra_session_secret: !Secret
-slog_token: !Secret
```
Comment (cat5inthecradle) [inline config/production.yml.erb:14]: I'm guessing there is cleanup to do in our secrets manager, but it shouldn't be breaking to remove this here.

File: dashboard/config/application.rb [hunk 2]
```diff
@@ -1,4 +1,3 @@
-require File.expand_path('../deployment', __FILE__)
```
Comment (cat5inthecradle) [inline dashboard/config/application.rb:1]: I believe this is referencing the deleted 'deployment.rb' file.

File: docs/where-are-the-logs.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Table below is horizontally scrollable, even if github's markdown renderer doesn
...
 | test        | email <br/> To: `dev+build@code.org` <br/> Subject: `/home/ubuntu/test/aws/build_and_mail_log test-websites`  | `ssh ubuntu@test.code.org`<br/>`~/test/code-dot-org/dashboard/test/ui/*.log`<br/>also on Sauce Labs at https://saucelabs.com/beta/dashboard/tests | `ssh ubuntu@test.code.org`<br/> `~/test/code-dot-org/pegasus/log/*.log`       | `ssh ubuntu@test.code.org` <br/>`~/test/code-dot-org/dashboard/log/test.log`          |
-| production  | email <br/>To: `dev+build@code.org` <br/>Subject: `/home/ubuntu/production/aws/build_and_mail_log websites` |                                                                                                                                      | on logentries: https://logentries.com                                    | on logentries: https://logentries.com                                            |
```
Comment (cat5inthecradle) [inline docs/where-are-the-logs.md:12]: This file as a whole doesn't seem incredibly up-to-date and useful.

**Comment by davidsbailey** [comment]
now i am seeing this error:
```
Dave-MBP:~/src/cdo/dashboard (remove-slog-logentries)$ rake db:migrate
rake aborted!
LoadError: cannot load such file -- cdo/poste
/Users/dsb/src/cdo/dashboard/config/application.rb:1:in `<top (required)>'
/Users/dsb/src/cdo/dashboard/Rakefile:4:in `<top (required)>'
(See full trace by running task with --trace)
```

**Review by jamescodeorg — APPROVED** [review]
Changes looks good to me, but I think Dave found an issue.

File: dashboard/config/application.rb (hunk 2 above)
Comment (davidsbailey) [reply dashboard/config/application.rb:1]: I believe this needs to be replaced with
```suggestion
require File.expand_path('../../../deployment', __FILE__)
```
to fix the issue with loading cdo/poste on the next line

Comment (cat5inthecradle) [reply dashboard/config/application.rb:1]: Yep, i was just coming to that conclusion myself, thanks!

**Review by Hamms — APPROVED** [review]
This LGTM, and I'm not able to reproduce the error Dave reported; Dave, are you able to repro even when running rake with `bundle exec`?

Thanks for tracking down all these instances we can remove, Darin! Any chance we're also now able to remove our dependency on the `le` gem? https://github.com/code-dot-org/code-dot-org/blob/419659ee9ff78e34f51447b40ea3f7250c96e089/Gemfile#L32

**Pull request merged**